front/src/utils/helpers.ts

```diff
@@ -0,0 +1,84 @@
+import { useState, useEffect } from 'react';
+import 'moment/locale/fr';
+
+/**
+ * Fonctions utiles pour les cartes des différents projets
+ */
+
+/**
+ * Debounce input fields
+ */
+export const useDebounce = (value: string | number, delay: number) => {
```

Comment:
Here you can use a special type to avoid casting
```suggestion
export const useDebounce = <T = string | number>(value: T, delay: number): T => {
```

Reply:
need to talk about it